.circleci/config.yml

```diff
@@ -3,13 +3,13 @@ version: 2
 jobs:
   build:
     docker:
-      - image: hashicorp/terraform:0.9.11
+      - image: hashicorp/terraform:0.10.1
```

Comment:
if we're adding a requirement on the 0.10.x series (instead of 0.9.x), this probably merits a new major release of `tf_aws_ecs`